sbpy/units/core.py

```diff
@@ -133,3 +136,150 @@ def spectral_density_vega(wfb):
         (flam0.unit, VEGA, lambda x: x / flam0.value,
          lambda x: x * flam0.value)
     ]
+
+
+@u.quantity_input(cross_section='km2', reflectance='1/sr',
+        f_sun=['W/(m2 um)', 'W/(m2 Hz)'])
+def reflectance(cross_section=None, reflectance=None, wfb=None, M_sun=None,
+        f_sun=None):
+    """Reflectance related equivalencies.
+
+    Support conversion from/to reflectance and scattering cross-section
+    to/from total flux and magnitude.
+
+    The default magnitude system is ``VEGAmag``, where the apparent magnitude
+    of Vega is assumed to be 0 in all and any wavelengths and bands.  If other
+    magnitude system is used, then it is implicitly inferred from the solar
+    magnitude passed by keyword 'M_sun'.
```

Comment:
"passed by keyword `M_sun`." i.e., change the quotes.

Reply:
Shouldn't the magnitude system be explicitly specified in the `.to` method?

Reply:
But there is no way for equivalency function to know what's passed to `.to`.

Reply:
I don't think there is any default magnitude system, but just a default bandpass and solar model.  I can use this to convert a magnitude in the ABmag system without specifying M_sun.  This is because f_sun_nu is defined.  I think the note should instead be: "The default bandpass is Johnson V and solar model is XXX.  If the user does not specify a wavelength, frequency, or bandpass via `wfb`, then VEGAmag and spectral flux density equivalencies will be provided for the V-band."  If you want to say more: "Users wanting conversions with magnitude systems incompatible with these units will need to define `M_sun` in that system or add equivalencies to convert from the magnitude system to physical units or Vega-based magnitudes."

Reply:
Now it makes sense to me.  Thanks for the suggestion.